src/pages/encrypt-hard-drive/encrypt-hard-drive.js

```diff
@@ -15,7 +15,7 @@ module.exports = React.createClass({
   },
   render: function() {
     return (
-      <div className="encrypt-hard-drive-page">
+      <div className="encrypt-hard-drive-page main-encrypt-page">
```

Comment:
css specificity was killing me, so I needed to control if it was the thank you page or not.